Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -357,7 +357,7 @@ public string MakeValidUsername(string username)
                 IncrementInvalidLoginAttemptCount(user);
             }
 
-            _logger.LogInformation("Authentication request for {0} {1}.", user.Name, success ? "has succeeded" : "has been denied");
+            _logger.LogInformation("Authentication request for {0} {1} (IP: {2}).", user.Name, success ? "has succeeded" : "has been denied", remoteEndPoint);
```

Comment:
The paramaters in the log template string should be specified like `{UserName}`, `{Success}`, etc (see [docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.loginformation?view=dotnet-plat-ext-3.1#Microsoft_Extensions_Logging_LoggerExtensions_LogInformation_Microsoft_Extensions_Logging_ILogger_Microsoft_Extensions_Logging_EventId_System_Exception_System_String_System_Object___)).